x/gamm/genesis.go

```diff
@@ -14,6 +14,7 @@ import (
 func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState, unpacker codectypes.AnyUnpacker) {
 	k.SetParams(ctx, genState.Params)
 	k.SetNextPoolNumber(ctx, genState.NextPoolNumber)
+	k.SetNextTwapHistoryDeleteIndex(ctx, genState.NextTwapHistoryDeletePoolId)
```

Comment:
Should set it to `0` or smth if genstate is blank? I have no idea what to set it to atm

Reply:
Do you mean the default genesis state? The default genesis state is 
```
func DefaultGenesis() *GenesisState {
	return &GenesisState{
		Pools:                       []*codectypes.Any{},
		NextPoolNumber:              1,
		Params:                      DefaultParams(),
		NextTwapHistoryDeletePoolId: 1,
	}
}
```

Reply:
Oh cool